Error fetching data from http://65.108.233.71:8080/v1: HTTPConnectionPool(host='65.108.233.71', port=8080): Max retries exceeded with url: /v1 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fc03c6deb90>, 'Connection to 65.108.233.71 timed out. (connect timeout=5)'))
{'http://73.181.115.53:8080/v1': 2440278, 'http://70.15.242.6:8080/v1': 2440280, 'http://65.109.80.179:8080/v1': 2440282, 'http://65.108.230.116:8080/v1': 2440304, 'http://65.108.15.240:8080/v1': 2440310, 'http://51.159.100.217:8080/v1': 2440310, 'http://222.101.31.242:8080/v1': 2440312, 'http://172.104.211.8:8080/v1': 2440312, 'http://158.220.123.33:8080/v1': 2440312, 'http://142.132.217.22:8080/v1': 2440312, 'http://50.29.171.32:8080/v1': 1755416, 'http://94.130.22.86:8080/v1': 1755416}
Nginx configuration updated!
